pkg/doc/verifiable/example_credential_test.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable_test
+
+import (
+	"crypto/ed25519"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable"
+)
+
+type UniversityDegree struct {
+	Type       string `json:"type,omitempty"`
+	University string `json:"university,omitempty"`
+}
+
+type UniversityDegreeSubject struct {
+	ID     string           `json:"id,omitempty"`
+	Name   string           `json:"name,omitempty"`
+	Spouse string           `json:"spouse,omitempty"`
+	Degree UniversityDegree `json:"degree,omitempty"`
+}
+
+type UniversityDegreeCredential struct {
+	*verifiable.Credential
+
+	ReferenceNumber int `json:"referenceNumber,omitempty"`
+}
+
+func (udc *UniversityDegreeCredential) MarshalJSON() ([]byte, error) {
+	// todo too complex! (https://github.com/hyperledger/aries-framework-go/issues/847)
+	c := udc.Credential
+	cp := *c
+
+	cp.ExtraFields = map[string]interface{}{
+		"referenceNumber": udc.ReferenceNumber,
+	}
+
+	return json.Marshal(&cp)
+}
+
+//nolint:gochecknoglobals
+var privKey = ed25519.PrivateKey{56, 237, 176, 143, 247, 162, 167, 111, 85, 161, 158, 14, 243, 173, 144, 51, 157, 109, 155, 228, 77, 170, 238, 85, 220, 144, 158, 51, 14, 40, 153, 141, 193, 179, 12, 234, 125, 193, 60, 56, 198, 150, 80, 93, 30, 58, 14, 152, 205, 6, 50, 98, 125, 212, 65, 17, 15, 11, 230, 3, 226, 187, 7, 89} //nolint:lll
```

Comment:
Why we are creating the priv key this way ?

Reply:
If we generate the key, we will get different JWS each time we are running the example. And we are checking the JWS output (2-nd **Outcome**). That's why I made the keys static.